docs/spelling_wordlist.txt

```diff
@@ -0,0 +1,51 @@
+AWS
+Autoscaling
+Genomics
+Indices
+OSX
+Slurm
+UTF
+Ubuntu
+alinux
+args
+autoscaling
+aws
+awsbhosts
+awsbkill
+awsbout
+awsbqueues
+awsbstat
+awsbsub
+berks
+config
+createami
+dir
+filesystem
+growroot
+hostname
+jobwatcher
+json
+keypair
+nodewatcher
+online
+parallelcluster
+pem
+pre
+qhost
+qstat
+qsub
+sge
+signalled
+slurm
```

Comment:
why also lowercase? and not SGE, Torque?

Reply:
they refer to config values that are lowercase but I'll move them to the spelling section for the specific file (like the other config values) rather than having them in the global dict.